addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -2622,7 +2627,9 @@ export class OdooEditor extends EventTarget {
             xy: {left: 'x', top: 'y'},
             size: {left: 'width', top: 'height'}
         };
-
+        const table = getInSelection(this.document, 'table');
+        const resetTableSize = ui.querySelector('.o_reset_table_size');
+        table && !table.hasAttribute('style') ? resetTableSize.classList.add('d-none') : resetTableSize.classList.remove('d-none');
```

Comment:
IMO ternary operators should only be used for variable assignation, not for side effects. This should be a proper if/else.